source/includes/cli-reference/_commands.html.md

```diff
@@ -74,44 +68,44 @@ BRAIN names may include:
 
 It is case insensitive, but case aware.
 
-## bonsai brain create
+## bonsai download
 
 ```
-$ bonsai brain create brainName
-```
-
-`create` generates a new brain and names it "brainName".  It also sets the assumed BRAIN name for later commands.
+$ bonsai download --help
+Usage: bonsai.py download [OPTIONS] BRAIN_NAME
 
-
-```
-$ bonsai brain create --help
-Usage: bonsai brain create [OPTIONS] BRAIN_NAME
-
-  Creates a BRAIN on the server.
+  Downloads all the files related to a BRAIN.
 
 Options:
   --help  Show this message and exit.
 ```
 
-[//]: # (There is no bonsai brain delete yet - CLI v2)
+`bonsai download` creates local copies of your BRAIN project files. This will contain your Inkling code and may also contain simulator code.
```

Comment:
Worth mentioning that `bonsai download` works like git clone - it copies files into a new directory, and will not try and overwrite files that already exist.